fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/OCFLPersistentStorageSession.java

```diff
@@ -177,6 +190,15 @@ public void persist(final ResourceOperation operation) throws PersistentStorageE
 
     }
 
+    private String mintOCFLObjectId ( final String fedoraIdentifier) {
+        //TODO make OBJECT minting more configurable.
+        String bareFedoraIdentifier = fedoraIdentifier;
+        if(fedoraIdentifier.indexOf(FEDORA_ID_PREFIX) == 0){
```

Comment:
This isn't path minting right, this is transforming a Fedora ID into an OCFL ID?

Reply:
Also, nit picky space `if (fed...`

Reply:
In some cases it may be a simple transaction.  In other cases it may be a minting.  It depends on what strategy the user decides to use: c.f. https://jira.lyrasis.org/browse/FCREPO-3156